src/components/PagePeople/PagePeople.tsx

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { PageTable } from '../PageTable/PageTable';
+
+export const PagePeople = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+  const [isLoaded, setIsLoaded] = useState(false);
+  const [hasError, setHasError] = useState(false);
+  const { personMan } = useParams();
```

Comment:
You may consider it using in the required component instead of passing it as an args to child component